src/battle/move/shell_crack/77F240.c

```diff
@@ -6,4 +6,24 @@
 
 #include "world/common/IsRightOnEquipped.inc.c"
 
-INCLUDE_ASM(s32, "battle/move/shell_crack/77F240", func_802A10A4_77F2E4);
+extern void fx_floating_cloud_puff(int, f32, f32, f32, f32, s32); /* extern */
+
+int func_802A10A4_77F2E4(void) {
```

Comment:
This is almost certainly a script API function (judging from the `return 2`), so please make its signature the following
```suggestion
ApiStatus func_802A10A4_77F2E4(Evt* script, s32 isInitialCall) {
```